addOns/graphql/src/test/java/org/zaproxy/addon/graphql/GraphQlParamUnitTest.java

```diff
@@ -122,4 +128,43 @@ void shouldUseDefaultWithInvalidRequestMethodOption() {
         // Then
         assertThat(options.getRequestMethod(), is(equalTo(RequestMethodOption.POST_JSON)));
     }
+
+    @Test
+    void shouldWriteConfigCorrectly() {
+        // Given
+        options = new GraphQlParam();
+        config = new ZapXmlConfiguration();
+        // When
+        options.load(config);
+        options.setQueryGenEnabled(false);
+        options.setMaxQueryDepth(1337);
+        options.setLenientMaxQueryDepthEnabled(false);
+        options.setMaxAdditionalQueryDepth(42);
+        options.setMaxArgsDepth(21);
+        options.setOptionalArgsEnabled(false);
+        options.setArgsType(ArgsTypeOption.INLINE);
+        options.setQuerySplitType(QuerySplitOption.ROOT_FIELD);
+        options.setRequestMethod(RequestMethodOption.POST_GRAPHQL);
+        options.setCycleDetectionMode(CycleDetectionModeOption.EXHAUSTIVE);
+        options.setMaxCycleDetectionAlerts(9999);
+        // Then
+        assertThat(
+                ConfigurationUtils.toString(config),
+                is(
+                        equalTo(
+                                """
+graphql.queryGenEnabled=false
+graphql.maxQueryDepth=1337
+graphql.lenientMaxQueryDepth=false
+graphql.maxAdditionalQueryDepth=42
+graphql.maxArgsDepth=21
+graphql.optionalArgs=false
+graphql.argsType=INLINE
+graphql.querySplitType=ROOT_FIELD
+graphql.requestMethod=POST_GRAPHQL
+graphql.cycleDetectionMode=EXHAUSTIVE
+graphql.cycleDetectionMaxAlerts=9999
+graphql[@version]=2\
```

Comment:
Is there really a trailing slash? Or is that leftover from a previous escaped quote?

Reply:
This was to escape the newline to allow adding the triple quotes to the next line, but we don't really need it. Tweaked.